frontend/src/component/feature/StrategyTypes/RolloutSlider/RolloutSlider.tsx

```diff
@@ -142,20 +195,47 @@ const RolloutSlider = ({
                     }
                 />
             </StyledBox>
-            <StyledSlider
-                min={0}
-                max={100}
-                value={value}
-                getAriaValueText={valuetext}
-                aria-labelledby='discrete-slider-always'
-                step={1}
-                data-testid={ROLLOUT_SLIDER_ID}
-                marks={marks}
-                onChange={onChange}
-                valueLabelDisplay='on'
-                disabled={disabled}
-            />
-        </div>
+            <StyledSliderContainer>
+                <SliderContent>
+                    <StyledSlider
+                        min={0}
+                        max={100}
+                        value={value}
+                        getAriaValueText={valuetext}
+                        aria-labelledby='discrete-slider-always'
+                        step={1}
+                        data-testid={ROLLOUT_SLIDER_ID}
+                        marks={marks}
+                        onChange={onChange}
+                        valueLabelDisplay='on'
+                        disabled={disabled}
+                    />
+                </SliderContent>
+                <StyledInputBox>
+                    <StyledTextField
+                        size='small'
```

Comment:
This input definitely needs a label. We can reuse the same one we use for the slider, though.

But also, unless that ID is used by something else, I'd suggest replacing it with `react.useId` to guarantee that it's unique even if there's multiple rollout sliders on a page. (don't know if that can happen atm? change requests? but best to be sure anyway.)

```suggestion
                        size='small'
                        aria-labelledby='discrete-slider-always'
```